spatiotemporal_exploration/src/spatiotemporal_exploration.cpp

```diff
@@ -795,7 +795,7 @@ int main(int argc,char* argv[])
     //to get relevant nodes
     nodeListClient = n.serviceClient<strands_navigation_msgs::GetTaggedNodes>("/topological_map_manager/get_tagged_nodes");
     //to create task objects
-    taskAdder = n.serviceClient<strands_executive_msgs::AddTask>("/task_executor/add_task");
+    taskAdder = n.serviceClient<strands_executive_msgs::AddTask>("/robot_routine/add_tasks");
```

Comment:
If you specify without the initial slash we can the use topic remapping to change this.